app/views/escorts/confirm_cancel.html.slim

```diff
@@ -0,0 +1,17 @@
+= content_for :breadcrumbs do
```

Comment:
Isn't a bit odd that we now have two confirmation pages (re-printing - 🤣 🏃 I still can take this a real feature -  and cancelling) and they are managed through the controller/views in a completely different way? Seems like instead of bringing consistency to the app, we're doing the opposite.